packaging/src/docker/docker-compose.yml

```diff
@@ -63,6 +70,10 @@ services:
     restart: unless-stopped
     container_name: hiveserver2
     environment:
+      HADOOP_CLASSPATH: /opt/hadoop/share/hadoop/tools/lib/*
+      HIVE_OPTS: >
+        -hiveconf fs.s3a.access.key=${AWS_ACCESS_KEY_ID}
```

Comment:
Will these configs be passed to tez containers?
As `fs.s3a.access.key` and `fs.s3a.secret.key` are these are part of hidden conf. As a workaround hadoop support bucket name in conf i.e. `fs.s3a.bucket.BUCKETNAME.access.key` and that's how I've been using internally.

I don't use docker images much, hence not sure if tez job is running within HS2 JVM, if its separate JVM then conf won't be passed for sure.

https://github.com/apache/hive/blob/79924c5bf3cfdb0df144b3364097e310bc2689b1/ql/src/java/org/apache/hadoop/hive/ql/exec/tez/DagUtils.java#L1476